### DISCUSSION TITLE
storing textbox values

### DISCUSSION THREAD

**Discussion by unknown-kel** [TomSchimansky/CustomTkinter#1493] [General]
How can i store textbox values in a variable, and access it in diffrent page.

**Comment by bfedie518** [comment]
You can't use a StringVar like in an Entry.  You have to retrieve the contents of the textbox with the [.get()](https://github.com/TomSchimansky/CustomTkinter/wiki/CTkTextbox#getindex1-index2none) method.

**Reply by unknown-kel** [reply]
I used .get() and stored the value in a variable, and made that variable global and am trying to access it in another frame
